starknet_py/net/schemas/rpc.py

```diff
@@ -111,13 +111,19 @@ class TransactionSchema(Schema):
 
 
 class InvokeTransactionSchema(TransactionSchema):
-    contract_address = Felt(data_key="contract_address", required=True)
-    entry_point_selector = Felt(data_key="entry_point_selector", required=True)
+    contract_address = Felt(data_key="contract_address", default=None)
+    sender_address = Felt(data_key="sender_address", default=None)
+    entry_point_selector = Felt(data_key="entry_point_selector", default=None)
```

Comment:
What's the difference between `load_default=None` and `default=None`?

Reply:
Not sure, but we should use `load_default` like everywhere